### ISSUE TITLE
Results Page - Add High and low bars for Cost Summary Graphical View

### ISSUE THREAD

**Issue by barrc** [Eastern-Research-Group/SWC#36]
@JasonBernagros and I discussed this.  I think the plan we settled on was:
- In the short term, add text to the plot that indicates the shown value is the median estimated cost and to see the tabular view for high/low estimates
- In the long run, consider using a mixed type chart to show the low, median, and high value

**Comment by barrc** [comment]
The desktop app looks like this:
![desktop-costs](https://user-images.githubusercontent.com/13906772/49679130-f4b12800-fa3d-11e8-8084-66fb9c8f73d4.jpg)

**Comment by JasonBernagros** [comment]
Colleen would you be able to include the type of line graphics that can be added in the web app for this graph to show the error bar values? I agree in the short term we should indicate that the median value of the costs is included in the bar graph and to refer to the tabular view to show high and low costs.

From the documentation (http://jtblin.github.io/angular-chart.js/) it doesn’t seem like the library we’re using supports error bars.  I could be misunderstanding since I am new to this library.

One alternative could be using a “mixed type chart” (at the bottom of the docs) with two line charts (one for high values and low values) overlaying the bar chart.  We can probably make the line charts unconnected.  It might be messy with both a baseline scenario and a current scenario.

In the short term I would probably recommend pointing users to the table since the information of high/low values is there, and maybe adding some text to the plot indicating that this is the median estimated cost and to see the tabular view for high/low estimates.  I know that’s not ideal so if this is a high priority I can spend more time researching it or try to implement the “mixed type chart” solution to see how it looks, or let me know if you have another idea.


​​
Jason T. Bernagros
Landscape Architect
LEED AP BD + C
US EPA Office of Research and Development
National Risk Management Research Laboratory
Water Systems Division, Water Resources Recovery Branch
O: Reagan Building, Desk 71263
P: 202-566-1671
E: bernagros.jason@epa.gov<mailto:bernagros.jason@epa.gov>
Mail: 1200 Pennsylvania Ave. NW, MC 8623R, Washington, DC  20460
Delivery: 1300 Pennsylvania., NW (Reagan Building, Desk 71263), Washington, DC  20004
W: (Green Infrastructure) http://www2.epa.gov/water-research/green-infrastructure-research
      (Water Innovation & Technology) https://www.epa.gov/water-innovation-tech

**Comment by barrc** [comment]
I thought it might work to adapt the example mixed type chart from the documentation linked above:

![image](https://user-images.githubusercontent.com/13906772/49893163-a1edbc80-fdff-11e8-8d91-60f04a440405.png)

**Comment by JasonBernagros** [comment]
I agree Colleen. I think each day would be labeled as an LID control with the bar graph showing the high and low values of the costs.

​​
Jason T. Bernagros
Landscape Architect
LEED AP BD + C
US EPA Office of Research and Development
National Risk Management Research Laboratory
Water Systems Division, Water Resources Recovery Branch
O: Reagan Building, Desk 71263
P: 202-566-1671
E: bernagros.jason@epa.gov<mailto:bernagros.jason@epa.gov>
Mail: 1200 Pennsylvania Ave. NW, MC 8623R, Washington, DC  20460
Delivery: 1300 Pennsylvania., NW (Reagan Building, Desk 71263), Washington, DC  20004
W: (Green Infrastructure) http://www2.epa.gov/water-research/green-infrastructure-research
      (Water Innovation & Technology) https://www.epa.gov/water-innovation-tech

**Comment by LiadovERG** [comment]
Need to check chart library 
10h
